diff --git a/node_modules/expo-localization/android/build.gradle b/node_modules/expo-localization/android/build.gradle
index 7e53d2b..2d6a1f1 100644
--- a/node_modules/expo-localization/android/build.gradle
+++ b/node_modules/expo-localization/android/build.gradle
@@ -1,8 +1,12 @@
-plugins {
-  id 'com.android.library'
-  id 'expo-module-gradle-plugin'
-}
+apply plugin: 'com.android.library'
 
+def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
+apply from: expoModulesCorePlugin
+applyKotlinExpoModulesCorePlugin()
+useCoreDependencies()
+useDefaultAndroidSdkVersions()
+useExpoPublishing()
+
 group = 'host.exp.exponent'
 version = '17.0.8'
 
